morebits.js

```diff
@@ -3614,10 +3654,36 @@ Morebits.wiki.page = function(pageName, status) {
 		var page = response.pages[0], rev;
 		ctx.pageExists = !page.missing;
 		if (ctx.pageExists) {
-			rev = page.revisions[0];
-			ctx.lastEditTime = rev.timestamp;
-			ctx.pageText = rev.content;
 			ctx.pageID = page.pageid;
+			ctx.lastTouchedTime = page.touched; // Used as basetimestamp when saving
```

Comment:
Noted in the commit summary, but I've changed to using `touched` rather than `rev.timestamp`: `revert` will still need to use `touched`, and it's probably better anyway.  It also means we don't need to query for `rvprop=timestamp` ever.

That being said, we *could* keep `timestamp` for `revert`, and include a separate getter for that, to go alongside e.g. `getRevisionUser`.

It does make `getLastEditTime` slightly incorrect or misleasing, but does that matter?  I suppose we could default to `rev.timestamp` for non-`revert` modes, then use `touched` just for revert`, but that seems worse.